File: youlexuanmanagerweb/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {

    this.search = function (page,rows,searchEntity) {
        return $http.post('../brand/search.do?page='+page+'&rows='+rows,searchEntity);
    }

    this.delete = function (selectIds) {
        return $http.get('../brand/delete.do?ids='+selectIds);
    }

    this.findOne = function (id) {
        return $http.get('../brand/findOne.do?id='+id);
    }

    this.save = function (methodName,entity) {
        return $http.post('../brand/'+methodName+'.do',entity);
    }
    
    
    this.selectOptionList = function () {
        return $http.get('../brand/selectOptionList.do');
    }

});

File: youlexuansellergoodsinterface/src/main/java/com/youlexuan/sellergoods/service/BrandService.java
package com.youlexuan.sellergoods.service;

import com.youlexuan.entity.PageResult;
import com.youlexuan.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {

    public List<TbBrand> findAll();
    public PageResult findPage(int pageNum, int pageSize);
    public void add(TbBrand brand);
    public TbBrand findOne(Long id);
    public void update(TbBrand brand);
    public void delete(Long[] ids);

    public PageResult search(TbBrand brand,int page,int rows);

    public List<Map> selectOptionList();
}
